x/projects/keeper/project_test.go

```diff
@@ -57,11 +57,66 @@ func TestCreateProject(t *testing.T) {
 
 	testkeeper.AdvanceEpoch(ctx, keepers)
 
-	// create another project with the same name, should fail as this is unique
-	err = keepers.Projects.CreateProject(sdk.UnwrapSDKContext(ctx), subAccount.Addr.String(), projectData, plan)
+	// test invalid project name/description
+	defaultProjectName := types.ADMIN_PROJECT_NAME
+	longProjectName := strings.Repeat(defaultProjectName, types.MAX_PROJECT_NAME_LEN)
```

Comment:
it doesn't matter since the defaultProjectName's length is bigger than 1. Assuming MAX_PROJECT_NAME_LEN = 50 and defaultProjectName = "admin" then len(longProjectName) = 50*5 which is way more than 50. But I'll add the +1 for readability

Reply:
indeed both tests should trigger the error, but the one with LEN+1 is tighter and therefore better: if a bug in the validation allows for LEN+1 then the current test won't detect it.